src/components/app/data/utils.js

```diff
@@ -56,6 +56,33 @@ export function isSystemMaintenanceAlertOpen(config) {
   return true;
 }
 
+export const hasActiveLicenseOrLicenseRequest = ({ subscriptionPlan, subscriptionLicense, licenseRequests }) => (
+  (
+    subscriptionPlan?.isCurrent && subscriptionLicense?.status === LICENSE_STATUS.ACTIVATED
+  ) || licenseRequests.length > 0
+);
+
+export const hasAssignedCodesOrCodeRequests = ({ couponCodesCount, couponCodeRequests }) => (
+  couponCodesCount > 0 || couponCodeRequests.length > 0
+);
+
+export const hasAutoAppliedLearnerCreditPolicies = ({ redeemableLearnerCreditPolicies }) => {
+  const autoAppliedPolicyTypes = [
+    POLICY_TYPES.PER_LEARNER_CREDIT,
+    POLICY_TYPES.PER_ENROLLMENT_CREDIT,
+  ];
+  return !!redeemableLearnerCreditPolicies.redeemablePolicies.filter(
+    policy => autoAppliedPolicyTypes.includes(policy.policyType),
+  ).length > 0;
+};
+
+export const hasAllocatedOrAcceptedAssignments = ({ redeemableLearnerCreditPolicies }) => (
```

Comment:
nit: if there's only 1 arg to a function, can probably avoid passing an object, and simply pass the underlying value itself (e.g., `hasAllocatedOrAcceptedAssignments(redeemableLearnerCreditPolicies)`)